src/_P028_BME280.ino

```diff
@@ -126,9 +129,9 @@ boolean Plugin_028(uint8_t function, struct EventStruct *event, String& string)
         static_cast<P028_data_struct *>(getPluginTaskData(event->TaskIndex));
 
       if (nullptr != P028_data) {
-        if (P028_data->sensorID != Unknown_DEVICE) {
+        if (P028_data->sensorID != P028_data_struct::Unknown_DEVICE) {
```

Comment:
Doesn't PVS Studio suggest to combine these `if` statements? Or do we leave that to the optimization phase of the compiler?

Reply:
Nested if statements will be optimized by the compiler.
What PVS Studio does (often) suggest is to make complex compares simpler.
Especially if the same boolean value is used on opposite sides of a logic operator.

Something like these:
```
if (A || (!A && B))
```

Very simple to do optimizations will be taken care of by the compiler.
```
if (A) {
  if (B) {
    ...
  }
}
```
Will result in the same compiled output as this:
```
if (A && B) {
  ...
}
```

Only exception might be when there are some variables defined within the inner scope.